developer/server/src/win32-tray.ts

```diff
@@ -81,7 +80,7 @@ module.exports = class Win32Tray {
     });
 
     //console.log('starting tray icon');
-    this.signals.forEach(signal => process.on(signal, this.shutdown.bind(this)));
+    process.on('exit', this.shutdown.bind(this));
```

Comment:
TSC was having trouble with mismatching types on the `signals` array. Given we only use it in one place, this was easier than trying to tidy it up further for now.